packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -134,6 +162,9 @@ class MarkdownBuilder implements md.NodeVisitor {
   /// Call when build an image widget.
   final MarkdownImageBuilder? imageBuilder;
 
+  /// Call when build an image widget.
```

Comment:
The docs for the old and new method should not be identical; both are currently inaccurate since there is no situation in which both are called, yet each claims to be called unconditionally.